CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 ### Fixed
 - core: Fix parsing of numeric literals in rule files
 - Java: fix the range and autofix of Cast expressions (#3669)
+- Generic mode scanner no longer tries to open submodule folders as files (#3690)
```

Comment:
You may want to change this reference to #3660 (original issue) or #3701 (new PR).

Reply:
```suggestion
- Generic mode scanner no longer tries to open submodule folders as files (#3701 )
```

Reply:
thanks for the fix!